plugin/CactbotEventSource/FFXIVProcessIntl.cs

```diff
@@ -698,5 +700,26 @@ public struct SamuraiJobMemory {
         }
       }
     }
+
+    [StructLayout(LayoutKind.Explicit)]
+    public struct SageJobMemory {
+      [FieldOffset(0x00)]
+      public ushort addersgallGauge; // the addersgall gauge, from 0 to 19999.
+
+      [FieldOffset(0x02)]
+      public byte addersgall;
+
+      [FieldOffset(0x03)]
+      public byte addersting;
+
+      [FieldOffset(0x04)]
+      private byte eukrasia_bits;
```

Comment:
Are the other bits used for anything, or can you pass the full byte? If you need to do the mask, can you mark this field as `[NonSerialized]`?

Reply:
I did only found that when you have an eukrasia, it is 1, otherwise 0.
I don't really know C# and even don't know what it is, do you mean like this:
```suggestion
      [NonSerialized]
      [FieldOffset(0x04)]
      private byte eukrasia_bits;
```